examples/s2_field_combination_extern_rf_train/nrw_crop_extern_s2_workflow_to_datasets_with_timeshift.ipynb

```diff
@@ -0,0 +1,2364 @@
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "points_source_operator = ge.unstable.workflow_operators.OgrSource(str(points_id))\n",
+    "points_source_operator.to_workflow_dict()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we create a workflow that joins the monthly Sentinel-2 data to the points of interest for the queried time-interval. This uses the `RasterVectorJoin` operator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'type': 'Vector',\n",
+       " 'operator': {'type': 'RasterVectorJoin',\n",
+       "  'params': {'names': ['B02', 'B03', 'B04', 'B08', 'NDVI'],\n",
+       "   'temporalAggregation': 'none',\n",
+       "   'featureAggregation': 'mean'},\n",
+       "  'sources': {'vector': {'type': 'OgrSource',\n",
+       "    'params': {'data': {'type': 'internal',\n",
+       "      'datasetId': '829fbbf7-5445-4f31-8deb-720885f74bc8'},\n",
+       "     'attributeProjection': None,\n",
+       "     'attributeFilters': None}},\n",
+       "   'rasters': [{'type': 'GdalSource',\n",
+       "     'params': {'data': {'type': 'internal',\n",
+       "       'datasetId': 'cf14cbe6-d747-48ca-abfe-bd9ea87618a4'}}},\n",
+       "    {'type': 'GdalSource',\n",
+       "     'params': {'data': {'type': 'internal',\n",
+       "       'datasetId': 'a22225d9-1e96-4646-ad67-0efe41291361'}}},\n",
+       "    {'type': 'GdalSource',\n",
+       "     'params': {'data': {'type': 'internal',\n",
+       "       'datasetId': '47d9ee89-6d87-4228-824d-46c52c751db0'}}},\n",
+       "    {'type': 'GdalSource',\n",
+       "     'params': {'data': {'type': 'internal',\n",
+       "       'datasetId': '3db91688-19f2-4dca-9995-0f4097b3996a'}}},\n",
+       "    {'type': 'GdalSource',\n",
+       "     'params': {'data': {'type': 'internal',\n",
+       "       'datasetId': '2a36bca3-fbe9-4d24-854f-fa159b755bfb'}}}]}}}"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "points_with_s2_cloud_free = ge.unstable.workflow_operators.RasterVectorJoin(\n",
+    "    raster_sources=[ge.unstable.workflow_operators.GdalSource(x) for x in monthly_band_data_id_map.values() ],\n",
+    "    vector_source=points_source_operator, #projected_points,\n",
+    "    new_column_names=[x for x in monthly_band_data_id_map.keys()],\n",
+    ")\n",
+    "\n",
+    "points_with_s2_cloud_free.to_workflow_dict()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since we want to train a Random-Forest on the monthly Sentinel-2 data, we need to attach all the months we are interested in to one point. The 'normal' way would be to query a time-interval and get points for each month with the respective values.\n",
```

Comment:
```suggestion
    "Since we want to train a Random-Forest on the monthly Sentinel-2 data, we need to attach all the months we are interested in into one feature. The 'normal' way would be to query a time interval and get points for each month with the respective values.\n",
```